docker-compose.yml

```diff
@@ -8,6 +8,9 @@ volumes:
 services:
   proton:
      image: ghcr.io/timeplus-io/proton:latest
+     ports:
+      - "3218:3218" # HTTP
```

Comment:
these 2 ports are not required to get data from owl-shop or cardemo. There are only needed for go/python drivers. So I don't think we need to declare them.

Reply:
I did not find anything from the readme to declare these port, it will be helpful to provide such sample